src/lib/dnssd/minimal_mdns/Server.h

```diff
@@ -77,11 +78,24 @@ class ServerBase
 public:
     struct EndpointInfo
     {
-        chip::Inet::InterfaceId interfaceId = chip::Inet::InterfaceId::Null();
-        chip::Inet::IPAddressType addressType;
-        chip::Inet::UDPEndPoint * listen_udp = nullptr;
 #if CHIP_MINMDNS_USE_EPHEMERAL_UNICAST_PORT
-        chip::Inet::UDPEndPoint * unicast_query_udp = nullptr;
+        EndpointInfo(chip::Inet::InterfaceId interfaceId, chip::Inet::IPAddressType addressType,
+                     chip::Inet::UDPEndPoint * listenUdp, chip::Inet::UDPEndPoint * unicastQueryUdp) :
+            mInterfaceId(interfaceId),
+            mAddressType(addressType), mListenUdp(listenUdp), mUnicastQueryUdp(unicastQueryUdp)
+        {}
+#else
+        EndpointInfo(chip::Inet::InterfaceId interfaceId, chip::Inet::IPAddressType addressType,
+                     chip::Inet::UDPEndPoint * listenUdp) :
+            mInterfaceId(interfaceId),
+            mAddressType(addressType), mListenUdp(listenUdp)
+        {}
+#endif
+        const chip::Inet::InterfaceId mInterfaceId;
```

Comment:
this is a struct: generally for structs we do not use the  `m*` naming.

Reply:
I changed the struct into a RAII style class.